tests/apm/overview.phpt

```diff
@@ -0,0 +1,310 @@
+--TEST--
+HHVM-190: APM Specification
+--FILE--
+<?php
+include dirname(__FILE__) . '/../utils.inc';
+
+$m = new MongoDB\Driver\Manager("mongodb://localhost:27017");
+cleanup( $m );
+
+class MySubscriber implements MongoDB\Driver\Monitoring\CommandSubscriber
+{
+	public function commandStarted( \MongoDB\Driver\Monitoring\CommandStartedEvent $event )
+	{
+		echo "started:\n";
+		var_dump( $event );
+	}
+
+	public function commandSucceeded( \MongoDB\Driver\Monitoring\CommandSucceededEvent $event )
+	{
+		echo "succeeded:\n";
+		var_dump( $event );
+	}
+
+	public function commandFailed( \MongoDB\Driver\Monitoring\CommandFailedEvent $event )
+	{
+		echo "failed:\n";
+		var_dump( $event );
+	}
+}
+
+$m->addSubscriber( new MySubscriber() );
+cleanup( $m );
+
+$d = new \MongoDB\BSON\Decimal128("1234.5678");
```

Comment:
I'd be in favor of removing the Decimal128 type here so that it's trivial to run this test against older MongoDB versions.